server/pages/index.js

```diff
@@ -488,7 +488,9 @@ function render404( request, response ) {
 	response.status( 404 ).send( renderJsx( '404', ctx ) );
 }
 
-function renderServerError( err, req, res ) {
+/* eslint-disable no-unused-vars */
+function renderServerError( err, req, res, next ) {
```

Comment:
Right, it's the signature that matters with these error handlers (previouly [broken](https://github.com/Automattic/wp-calypso/pull/26655#issuecomment-414378404)). Good catch!

Let's maybe add a comment to the code so future generations of developers :smile: touching this code will know _why_ there's an ununsed arg. Also, let's change the `eslint-disable`/`eslint-enable` pair to either `eslint-disable-line` (on the same line) or `eslint-disable-next-line` (on the previous line).

Reply:
I tried to use `eslint-disable-next-line` but prettier comes in the way and shifts it to the next line. I'll add a comment 👍